sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedRleValuesReader.java

```diff
@@ -407,7 +426,7 @@ public void skipIntegers(int total) {
 
   @Override
   public void skipBooleans(int total) {
-    throw new UnsupportedOperationException("only skipIntegers is valid");
+    skipIntegers(total);
```

Comment:
nit: maybe create another method like `skipN` and have `skipIntegers` and `skipBooleans` share it? it's a bit confusing when first reading this.

Reply:
OK ~

Reply:
@sunchao 7a9d4fd fix this